tools.yml

```diff
@@ -1118,12 +1118,30 @@ tools:
   toolshed.g2.bx.psu.edu/repos/genouest/braker3/braker3/.*:
     cores: 16
     mem: 61.4
+  toolshed.g2.bx.psu.edu/repos/goeckslab/mesmer/mesmer/.*:
+    cores: 24
+    mem: 478
+  toolshed.g2.bx.psu.edu/repos/goeckslab/rename_tiff_channels/rename_tiff_channels/.*:
+    cores: 1
+    mem: 2
+  toolshed.g2.bx.psu.edu/repos/goeckslab/scimap_mcmicro_to_anndata/scimap_mcmicro_to_anndata/.*:
+    cores: 4
+    mem: 16
+  toolshed.g2.bx.psu.edu/repos/goeckslab/scimap_phenotyping/scimap_phenotyping/.*:
+    cores: 4
+    mem: 16
+  toolshed.g2.bx.psu.edu/repos/goeckslab/vitessce_spatial/vitessce_spatial/.*:
+    cores: 8
+    mem: 64
   toolshed.g2.bx.psu.edu/repos/guerler/charts/charts/.*:
     mem: 10
   toolshed.g2.bx.psu.edu/repos/hammock/hammock/hammock_1.0/.*:
     mem: 20
     env:
       _JAVA_OPTIONS: -Xmx{int(mem)}G -Xms1G
+  toolshed.g2.bx.psu.edu/repos/imgteam/bfconvert/ip_convertimage/.*:
+    cores: 12
+    mem: 128
```

Comment:
This seems to be a lot, is this really needed? Is this dependent on the input size?

Reply:
I guess you haven't seen mesmer yet...

As I mentioned in the initial comment, the values come from the cancer.usegalaxy.org instance where the tools run correctly. So I figured we start there and bring it down if possible once we have some runtime data?

Reply:
OK, nice. So do you think we should go with 64GB or bring it all the way down to 24?

Reply:
At least this is what data suggesting, maybe you have large inputs and it takes more mem? or it scales with cores? In that case we need to have a nice rule.

Reply:
I have no usage data atm so I'll just drop it to 30GB for the time being.

Reply:
Forgot I didn't actually update the config following my reply... Updated now.